app/constants/calculate-workload-points-operation.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
Minor: The file name should be more appropriate to being constant, it appears to be a class/method.

Reply:
So named as the constant operations for calculate-workload-points